src/js/modules/core/components/ConfirmationPage.tsx

```diff
@@ -0,0 +1,308 @@
+  },
+  header: {
+    padding: "10px",
+    fontSize: theme.titleFontSize
+  },
+  form: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    fontSize: "1.0em",
+    paddingBottom: "40px"
+  },
+  inputs: {
+    display: "flex",
+    lineHeight: "2rem",
+    flexDirection: "column"
+  },
+  resumeUpload: {
+    maxWidth: "500px",
+    padding: "20px",
+    lineHeight: "1.8rem"
+  },
+  submit: {
+    width: "fit-content",
+    padding: "13px",
+    fontSize: "1.3rem",
+    border: "none",
+    maxWidth: "250px"
+  },
+  checkbox: {
+    width: "20px",
+    height: "20px"
+  },
+  radio: {
+    width: "20px",
+    height: "20px"
+  },
+  formItem: {
+    maxWidth: theme.formElementMaxWidth,
+    lineHeight: "1.8rem"
+  },
+  label: {
+    padding: "15px",
+    flex: 1
+  }
+});
+
+class ConfirmationPage extends React.Component<Props> {
+  handleSubmit = values => {
+    console.log('submit');
+    console.log(values);
+    //this.props.submitConfirmation(values, []);
+  };
+
+  validateForm = (values: FormData): Array<object> => {
+    if (values['location'] !== 'cannot-attend') {
+      const incomplete = getIncompleteFields(values, requiredFields);
+      return incomplete;
+    }
+    else {
+      return [];
+    }
+  };
+
+  render() {
+    let {
+      classes,
+      isConfirming,
+      user,
+      formData,
+      confirmTimestamp,
+      resumeTimestamp
+    } = this.props;
+    if (confirmTimestamp) {
+      return <SubmittedPage />;
+    } else {
+      return (
+        <div className={classes.ConfirmationPage}>
+          <h1 className={classes.header}>
+            Welcome!
+          </h1>
+          <p className={classes.p}>
+            Before you are confirmed for the event, there are a few things for you to read and fill out.
+            If this form is not filled out, you will not be eligible to attend HackNYU. Please read the following carefully:
+          </p>
+          <ul className={classes.ul}>
+            <li className={classes.li}>
+              At this time, participation at either the Abu Dhabi or Shanghai
+              location is <strong>only</strong> available for NYU students who
+              are currently enrolled at those campuses.
+            </li>
+
+            <li className={classes.li}>
+              Any student who is or has been enrolled in the last 12 months at a
+              high school or university can participate at our Brooklyn, NY
+              location.
+            </li>
+            <li className={classes.li}>
+              If you are under 18 years of age at the time of the event, you
+              must have your parent(s) or legal guardian(s) print and sign the
+              Minors Release Form, which you can access <a href="/pdf/minors-waiver.pdf" target="_blank"> here. </a>
+              Please be sure to bring a physical copy of this waiver to you when
+              you arrive to HackNYU, otherwise we will not be able to let you
```

Comment:
minor grammar issues:
```suggestion
              you arrive at HackNYU, otherwise we will not be able to let you
```